File: Iniciante/1040.py
N1, N2, N3, N4 = input().split()

N1 = float(N1)
N2 = float(N2)
N3 = float(N3)
N4 = float(N4)

M = ((N1*2)+(N2*3)+(N3*4)+(N4*1)) / 10

print(f'Media: {M:.1f}')

if(M>=7.0):
    print('Aluno aprovado.')

if(M<5.0):
    print('Aluno reprovado.')

if(M>=5.0 and M<=6.9):
    print('Aluno em exame.')
    NE = float(input())
    print(f'Nota do exame: {NE:.1f}')
    MF = (M + NE) / 2
    
    if(MF>=5.0):
        print('Aluno aprovado.')
    else:
        print('Aluno reprovado.')
    print(f'Media final: {MF:.1f}')

File: Iniciante/1040.c
#include <stdio.h>

int main(){

    float n1, n2, n3, n4, m, ne, mf;

    scanf("%f %f %f %f", &n1, &n2, &n3, &n4);
    m = (n1*2 + n2*3 + n3*4 + n4*1)/10;

    printf("Media: %.1f\n", m);

    if(m>=7.0){
        printf("Aluno aprovado.\n");
    }
    if(m<5.0){
        printf("Aluno reprovado.\n");
    }
    if(m>=5.0 && m<=6.9){
        printf("Aluno em exame.\n");
        scanf("%f", &ne);
        printf("Nota do exame: %.1f\n", ne);
        mf = (m + ne) / 2;

        if(mf>=5.0){
            printf("Aluno aprovado.\n");
        }else{
            printf("Aluno reprovado.\n");
        }
        printf("Media final: %.1f\n", mf);
    }

    return 0;
}